projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1067 @@
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+`available` `unavailable`
+
+**Example**
+
+Get all events that are not sold out or fully booked:
+
+```
+GET /events/?q=bookingAvailability:available
+```
+
+Get all places that are permanently closed:
+
+```
+GET /places/?q=bookingAvailability:unavailable
+```
+
+TO DO: link to booking availability guide when that guide is finished.
+
+### calendarType
+
+Every event and place in UiTdatabank has one of the following four calendarTypes:
+
+* `Single`: the event occurs on a single date, indicated by a single startDate and endDate.
+* `Multiple`: the event occurs on multiple dates, and has multiple subEvent entries with each a different startDate and endDate.
+* `Periodic`: the event or place runs for a specific period as indicated by its startDate and endDate, and can optionally have openingHours.
+* `Permanent`: the event or place is permanent and has no startDate or endDate, but it can optionally have openingHours.
+
+With the `calendarType` field you can look for results that match a certain calendarType.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers`
+
+**Possible values**
+
+`single` `multiple` `periodic` `permanent`
+
+**Example**
+
+Retrieve all permanent events and places:
+
+```
+GET /offers/?q=calendarType:permanent
+```
+
+### completedLanguages
+
+All documents created in UiTdatabank are available in Dutch (NL). Besides Dutch, content editors can use UiTdatabank to translate the name and description of their documents in French (FR), German (DE) and English (EN). The API supports translations in every [two-letter language ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
+
+You can use the `completedLanguages` field to only look for documents that are fully translated to a specific language.
+
+> Note that if an optional field is left empty in all languages, a translation can still be considered to be complete if all other fields that have a value are translated.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Allowed values for both are any of the two-letter language ISO 639-1 codes, but the most used languages are: `nl` `fr` `de` `en`
+
+**Example**
+
+Retrieve all organizers that are fully translated in English:
+
+```
+GET /organizers/?q=completedLanguages:en
+```
+
+### created
+
+UiTdatabank keeps track of the creation dates of each document. This metadata for the creation date can be queried with the `created` field.
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+Any ISO-8601 datetime, e.g. `2023-04-01T12:08:01+01:00`
+
+<!-- theme: info -->
+
+> Note that the `+` sign must be encoded for URLs (as `%2B`). Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.
```

Comment:
(Copied from older comment on previous filename)

I think technically everything inside the `q` must be URL-encoded.

For example when I go to [https://search.uitdatabank.be/offers/?q=labels:"UiTPAS Leuven"](https://search.uitdatabank.be/offers/?q=labels:%22UiTPAS%20Leuven%22) I can see that Firefox changes it to https://search.uitdatabank.be/offers/?q=labels:%22UiTPAS%20Leuven%22 "under the hood" (but in the URL bar it will not change as it will show it as "human-readable" to users).

So maybe something to mention at the start of the page (or in the escaping guide and call that for example "escaping & encoding"), and then mention here something like:

> Note that because everything inside `q` must be URL-encoded, the `+` sign should be encoded for as `%2B`. Otherwise it will be interpreted as whitespace and the given date time will be considered invalid.

Reply:
I will add this in the escaping (& encoding) guide